master/internal/core_checkpoint.go

```diff
@@ -175,6 +236,6 @@ func (m *Master) getCheckpoint(c echo.Context) error {
 		}
 	}
 
-	c.Response().Header().Set(echo.HeaderContentType, mimeType)
-	return m.getCheckpointImpl(c.Request().Context(), id, mimeType, c.Response())
+	c.Response().Header().Set(echo.HeaderContentType, MIMEApplicationZip)
+	return m.getCheckpointImpl(c.Request().Context(), id, MIMEApplicationZip, c.Response())
```

Comment:
It looks like you are disabling tgz-format downloads.  I don't understand why you had to change the mime type code at all?

Reply:
When the browser opens the download request, the browser's default request mime type is not zip or tgz so it got rejected. I'll change the logic here so we only set ZIP if the mime type is not already set.

Reply:
question: why not support either zip or tgz formats from the webui?  Why not let the user decide which format to download?